rest-api/src/main/java/org/jboss/pnc/rest/api/parameters/BuildParameters.java

```diff
@@ -36,19 +36,15 @@
     @DefaultValue("false")
     boolean temporaryBuild;
 
-    @Parameter(description = "Should we force the rebuild?")
-    @QueryParam("forceRebuild")
-    @DefaultValue("false")
-    boolean forceRebuild;
-
     @Parameter(description = "What should varant rebuild?")
     @QueryParam("rebuildMode")
+    @DefaultValue("IMPLICIT_DEPENDENCY_CHECK")
     RebuildMode rebuildMode;
 
     @Parameter(description = "Should we build also dependencies of this BuildConfiguration?")
     @QueryParam("buildDependencies")
     @DefaultValue("true")
-    boolean buildDependencies;
+    Boolean buildDependencies;
```

Comment:
what if it would be 
```suggestion
    boolean buildDependencies = true;
```
Would that work? Would that break something?

Reply:
Then you would always have buildDependencies=true in a request. I don't have a strong opinion on this.